### PR TITLE
hide scrollbar for databrowser

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?188390

code buddy @TylerSheffels

### CODE REVIEW

**Comment by millerdev** [comment]
Seems like this will work. Would it be better to fix the thing that's causing the content to be sized larger than the `.fd-accessory-pane` height? (seems like that's the real bug)

File: src/less-style/structure.less [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 .fd-accessory-pane {
   overflow-x: hidden;
-  overflow-y: auto;
+  overflow-y: hidden;
```
Comment (orangejenny) [inline src/less-style/structure.less]: super picky, but this could now be a single `overflow` rule

**Comment by millerdev** [comment]
GTM on test pass if fixing the content sizing would is too hard or if we want to get this in and come back to it later

**Comment by emord** [comment]
@millerdev Looks like it's only off by one pixel, but I haven't been able to find where that 1 px comes from. I'll look into it again later.

**Comment by emord** [comment]
Looks like it was because `line-height: 25px; height: 24 px`. Bumped up `height` to equal `line-height`

**Comment by millerdev** [comment]
Excellent! this is looking good

> it was because `line-height: 25px; height: 24 px`.

What do you think about using a variable to make the two the same, thereby making the linkage explicit? Something like `@lineHeight: 25px; line-height: @lineHeight; height: @lineHeight;`

Or maybe remove the `line-height` (or maybe `height`) rule altogether given that `height` is being set and assuming there will never be enough text to wrap to multiple lines?

Feel free to just merge if this is getting too nit-picky or you feel it's not worth the effort.

**Comment by emord** [comment]
Would have preferred just getting rid of one of the rules, but there's some funkiness with the minimize button if I do that (goes a little off center)

**Pull request merged**